<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleWidget</class>
 <widget class="QWidget" name="RoleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>455</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QGridLayout" name="role_grid">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="1" column="1">
    <widget class="QDateTimeEdit" name="validity_dte">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="displayFormat">
      <string>yyyy-MMM-dd hh:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::OffsetFromUTC</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="conn_limit_sb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="validity_chk">
     <property name="text">
      <string>Validity</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="conn_limit_lbl">
     <property name="font">
      <font>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Connections:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="attributes_gb">
     <property name="title">
      <string>&amp;Attributes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="superusr_chk">
        <property name="text">
         <string>Superuser</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="inh_perm_chk">
        <property name="text">
         <string>Inherit permissions</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="create_db_chk">
        <property name="text">
         <string>Can create database</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="bypass_rls_chk">
        <property name="text">
         <string>Bypass RLS</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="can_replicate_chk">
        <property name="text">
         <string>Can use replication</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="can_login_chk">
        <property name="text">
         <string>Can login</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="create_role_chk">
        <property name="text">
         <string>Can create role</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTabWidget" name="members_twg">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="member_of_tab">
      <attribute name="title">
       <string>&amp;Members</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="adm_members_tab">
      <attribute name="title">
       <string>&amp;Admin option</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="members_tab">
      <attribute name="title">
       <string>M&amp;ember of</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="passwd_lbl">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="passwd_edt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>passwd_edt</tabstop>
  <tabstop>validity_chk</tabstop>
  <tabstop>validity_dte</tabstop>
  <tabstop>conn_limit_sb</tabstop>
  <tabstop>superusr_chk</tabstop>
  <tabstop>inh_perm_chk</tabstop>
  <tabstop>create_db_chk</tabstop>
  <tabstop>create_role_chk</tabstop>
  <tabstop>can_login_chk</tabstop>
  <tabstop>can_replicate_chk</tabstop>
  <tabstop>bypass_rls_chk</tabstop>
  <tabstop>members_twg</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
